packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3022,6 +3022,8 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     MoveCursorHandler onMoveCursorForwardByCharacter,
     MoveCursorHandler onMoveCursorBackwardByCharacter,
     SetSelectionHandler onSetSelection,
+    VoidCallback onAccessibilityFocus,
```

Comment:
Same suggestion here about naming these "didGainAccessibilityFocus" and "didLoseAccessibilityFocus".